### PR TITLE
Add EKF2 debugging and wintunnel measurement

### PR DESCRIPTION
**Pull request by fvantienen**
- Add filename to logger status message
- Add more debugging information for EKF2
- Add message for wintunnel measurement

### CODE REVIEW

**Comment by fvantienen** [comment]
Can this be reviewed so that I can create a PR for Paparazzi with the correct pprzlink?

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -774,6 +774,7 @@
       <field name="status" type="uint8" values="STOPPED|RUNNING|ERROR">General status of the logger</field>
       <field name="errno" type="uint8">Error number, depend of the logging system, provides detailed information</field>
       <field name="used" type="uint32" unit="bytes">Accumulated number of bytes written by the logger</field>
+      <field name="filenames" type="char[]"/>
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: several files possible here ? please add a description.
also don't forget to update in paparazzi part the chibios logger that is using this message

Comment (fvantienen) [reply message_definitions/v1.0/messages.xml]: I will add the description. The update in paparazzi is on its way.

**Comment by fvantienen** [comment]
Resolved the comment

**Pull request merged**